cloud/blockstore/tests/csi_driver/test.py

```diff
@@ -250,3 +262,65 @@ def test_nbs_csi_driver_mounted_disk_protected_from_deletion():
         raise
     finally:
         cleanup_after_test(env)
+
+
+def test_nbs_csi_driver_volume_stat():
+    # Scenario
+    # 1. create volume and publish volume
+    # 2. get volume stats and validate output
+    # 3. create two files in the mounted directory
+    # 4. get volume stats again and validate output
+    # 5. check that the difference between used/available bytes is 2 block sizes
+    # 6. check that the difference between used/available inodes is 2
+    env, run = init()
+    try:
+        volume_name = "example-disk"
+        volume_size = 1024 ** 3
+        pod_name = "example-pod"
+        pod_id = "deadbeef"
+        env.csi.create_volume(name=volume_name, size=volume_size)
+        env.csi.publish_volume(pod_id, volume_name, pod_name)
+        stats1 = env.csi.volumestats(pod_id, volume_name)
+
+        assert "usage" in stats1
+        usage_array1 = stats1["usage"]
+        assert 2 == len(usage_array1)
+        for usage in usage_array1:
+            usage = usage_array1[0]
+            assert {"unit", "total", "available", "used"} == usage.keys()
+            assert 0 != usage["total"]
+            assert usage["total"] == usage["available"] + usage["used"]
+
+        mountPath = f"/var/lib/kubelet/pods/{pod_id}/volumes/kubernetes.io~csi/{volume_name}/mount"
```

Comment:
Using pathlib.Path is a better practice.

Reply:
done